Sources/MySQL/Database.swift

```diff
@@ -74,6 +78,7 @@ public final class Database {
     private let socket: String?
     private let flag: UInt
     private let encoding: String
+    public static var fractionalSecondsDigits: Int = 0
```

Comment:
I think that this would require making additional coupling between the Database, Connection and Bind classes.  I thought that, within the context of an application, it is very unlikely that the there would be two database instances and, if there was, it would be unlikely that they had different time resolutions.  

If I coupled these classes, would that be more acceptable?